test/spoom/cli/srb/coverage_test.rb

```diff
@@ -488,6 +492,45 @@ def test_finish_on_original_branch
           assert_equal("fake-branch", @project.git_current_branch)
         end
 
+        def test_config_options_string
+          sorbet_config = @project.read("sorbet/config")
+          # Add ignore path to config to test config options string
+          @project.write!("sorbet/config", "\n--ignore=lib/a.rb", append: true)
```

Comment:
Should we try with a wildcard `*` to show the expected behavior?

Reply:
The coverage tests are kind of expensive, do you mind merging both of them please?